doc/content/tutorials/openmc_mgxs.md

```diff
@@ -0,0 +1,202 @@
+extruded the mesh an additional 21.42 cm to cover the reflector. These changes are necessary to preserve mesh connectivity
+for deterministic transport solvers using these cross sections, and to ensure we generate cross sections for
+the reflector above the core.
+
+!listing /tutorials/lwr_mgxs/mesh.i
+  start=[3D_Core]
+  end=[Rename]
+
+The input mesh Cardinal will use to store cell tally results on (`mesh_in.e`) can be generated by running the
+command below. The resulting mesh can be found in [mgxs_mesh]; we note that this mesh is substantially finer
+in then a conventional Cardinal mesh mirror to adequately capture spatial gradients in the determistic solver.
+
+```bash
+cardinal-opt -i mesh.i --mesh-only
+```
+
+!media mgxs_mesh.png
+  id=mgxs_mesh
+  caption=The mesh mirror used for computing and storing [!ac](MGXS), shown with an isometric view and sliced on the $x-y$ plane
+  style=width:90%;margin-left:auto;margin-right:auto
+
+### Neutronics Input File id=neutronics
+
+The neutronics calculation is performed over the entire assembly by OpenMC and the wrapping of the OpenMC results in MOOSE is performed in
+`openmc_mgxs.i`. We begin by defining the mesh (generated in the previous step) used by Cardinal to determine which cell tallies should be constructed for [!ac](MGXS)
+generation:
+
+!listing /tutorials/lwr_mgxs/openmc_mgxs.i
+  block=Mesh
+
+Afterwards, we add two auxvariables and auxkernels for post-processing the cross sections -- we use these to compute the scattering ratio (the sum of scattering cross sections divided by the total cross section) in each energy group. The scattering ratio for each group should
+be less than or equal to one; scattering ratios larger than one may result in non-convergence of the deterministic solver or convergence to
+incorrect results. Note that we did not delete the gap blocks in the mesh, but we are block restricting the auxkernels and auxvariables such that
+they are not computed over the gap region (block 2). This is necessary as the gaps do not receive any tally hits in OpenMC. Therefore, the
+cross sections will be zero resulting in a divide by zero when computing scattering ratios.
+
+!listing /tutorials/lwr_mgxs/openmc_mgxs.i
+  block=AuxVariables AuxKernels
+
+Next, the [Problem block](Problem/index.md) (with an [OpenMCCellAverageProblem](OpenMCCellAverageProblem.md)) describes the syntax necessary to
+replace the normal MOOSE finite element calculation with an OpenMC neutronics solve. We select `cell_level = 1` to ensure that we generate our
+cell to element mapping for an OpenMC geometry depth one lower than the root universe. This will result in a distributed cell tally for each unique
+geometric region in the [!ac](LWR) lattice, as opposed to the entire lattice (which is what would happen with `cell_level = 0`). We then specify the number
+of particles (`particles = 10000`), the number of inactive batches (`inactive_batches = 50`), and the total number of batches (`batches = 1000`).
```

Comment:
```suggestion
of particles, the number of inactive batches, and the total number of batches.
```

Reply:
Justification: self-evident from the input file what name corresponds to each, and this insulates us if the input file param value ever changes.